lib/open_mic.rb

```diff
@@ -0,0 +1,44 @@
+class OpenMic
+  def initialize(event)
+    @event = {
+      location: event[:location],
+      date: event[:date],
+      performers: [],
+    }
+  end
+
+  def location
+    @event[:location]
+  end
+
+  def date
+    @event[:date]
+  end
+
+  def performers
+    @event[:performers]
+  end
+
+  def welcome(user)
+    @event[:performers] << user
+  end
+
+  def repeated_jokes?
```

Comment:
The logic here looks great!  One possible solution or refactor could be to keep track of jokes_told as an instance variable add a joke to it any time 'user.tell(user,joke)' is called, and then check the uniqueness of values in that array in the `repeated_jokes?` method